audits/rkm0959.md

```diff
@@ -250,7 +250,7 @@ Then, as we move "down" the recursion tree, we assert that
 
 This means that our `vk_root` check "propagates downwards" the tree.
 
-## 5. [Medium] `cumulative_sum` needs to be observed when we observe the `permutation_commit`
+## 5. [Medium] `cumulative sum` needs to be observed when we observe the `permutation_commit`
```

Comment:
Why did you remove the underscore here?  I think it's part of a variable name, and should be quoted as is.